packages/mint-components/src/components/sqm-dropdown-field/useDropdownField.tsx

```diff
@@ -0,0 +1,20 @@
+import { useDomContext } from "@saasquatch/stencil-hooks";
+const CONTEXT_NAME = "sq:validation-state";
+
+type FormState = {
+  error?: string;
+  validationErrors?: ValidationErrors;
+};
+
+type ValidationErrors = {
+  [key: string]: string;
+};
```

Comment:
These should be imported

```suggestion
```

Reply:
using exported types now + updated view types to match 👍